### PR TITLE
An array is a collection of hedgehogs

### PR DESCRIPTION
**Pull request by Tobski**
As per https://twitter.com/qntm/status/1105283419011977216

### CODE REVIEW

**Comment by Tobski** [comment]
Do not merge this. It is a painfully silly joke. Sorry 😬

File: chapters/textures.txt [hunk 1]
```diff
@@ -12,12 +12,12 @@ those instructions which take an code:OpTypeImage (representing a
...
 (sname:VkImageView, sname:VkSampler) pair) and texel coordinates as
 operands, and return a value based on one or more neighboring texture
-elements (_texels_) in the image.
+hedgehogs (_texels_) in the image.
 
 [NOTE]
 .Note
 ====
-Texel is a term which is a combination of the words texture and element.
+Texel is a term which is a combination of the words texture and hedgehog.
```
Comment (krOoze) [inline chapters/textures.txt]: LGTM

File: appendices/glossary.txt [hunk 2]
```diff
@@ -119,7 +119,7 @@ Binding (Memory)::
     An association established between a range of a resource object and a
     range of a memory object.
     These associations determine the memory locations affected by operations
-    performed on elements of a resource object.
+    performed on hedgehogs of a resource object.
```
Comment (krOoze) [inline appendices/glossary.txt]: PETA might object

**Comment by mikew-lunarg** [comment]
Needs an authoritative reference on the [Wikipedia](https://en.wikipedia.org/wiki/List_of_English_terms_of_venery,_by_animal) page.

**Comment by oddhack** [comment]
... you're not serious, are you?

**Pull request closed without merging**